docs/data/joy/components/aspect-ratio/aspect-ratio.md

```diff
@@ -1,6 +1,7 @@
 ---
 product: joy-ui
 title: React Aspect Ratio component
+packageName: @mui/joy
```

Comment:
Would it make sense to do 

```suggestion
```

 and instead to rely on a sane default with `product: joy-ui`?

Reply:
Done

Reply:
I would rather do this in separate PR